plugin/Example_Plugin.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @author shitake
+ * 
+ */
+
+var program = PluginManager.createPluginProgram('example');
+
+/**
+ * Plugin command: 
+ * 
+ * example true
+ * 
+ * output:
+ *  Plugin: Exmaple, Params: {isRun :true}
+ */
+program.command('isRun:boolean', function(params) {
+  console.log('Plugin: exmaple, Params: {isRun: ' + params.isRun + '}')
+})
+
+/**
+ * Plugin command: 
+ * 
+ * example init helloworld 12 12 640 480
+ * 
+ * output:
+ *  Plugin: Exmaple, Command: init, Params: {title: helloworld, x: 12, y: 12, width: 640, height: 480}
+ */
+program.command('init', 'title:string, x:number, y:number, width:number, height:number', function(params) {
```

Comment:
Why would you supply the parameters as a string here, when you just split them up at "," again later?
Wouldn't it make more sense to pass a dictionary right away, e.g. {"title":"string", "x":"number", "y":"number",...} ?